google/cloud/bigtable/internal/rpc_policy_defaults.h

```diff
@@ -0,0 +1,114 @@
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef GOOGLE_CLOUD_CPP_GOOGLE_CLOUD_BIGTABLE_INTERNAL_RPC_POLICY_DEFAULTS_H_
+#define GOOGLE_CLOUD_CPP_GOOGLE_CLOUD_BIGTABLE_INTERNAL_RPC_POLICY_DEFAULTS_H_
+
+#include "google/cloud/bigtable/version.h"
+#include <chrono>
+
+namespace google {
+namespace cloud {
+namespace bigtable {
+inline namespace BIGTABLE_CLIENT_NS {
+namespace internal {
+
+struct RPCPolicyDefaults {
+  std::chrono::milliseconds initial_delay;
+  std::chrono::milliseconds maximum_delay;
+  std::chrono::milliseconds maximum_retry_period;
+};
+
+// Interface name: "google.bigtable.admin.v2.BigtableInstanceAdmin"
+// Define the defaults using a pre-processor macro, this allows
+// the application developers to change the defaults for their
+// application by compiling with different values.
+
+#ifndef BIGTABLEINSTANCEADMIN_DEFAULT_INITIAL_DELAY
+#define BIGTABLEINSTANCEADMIN_DEFAULT_INITIAL_DELAY std::chrono::milliseconds(5)
+#endif  // BIGTABLEINSTANCEADMIN_DEFAULT_INITIAL_DELAY
+
+#ifndef BIGTABLEINSTANCEADMIN_DEFAULT_MAXIMUM_RETRY_PERIOD
+#define BIGTABLEINSTANCEADMIN_DEFAULT_MAXIMUM_RETRY_PERIOD \
+  std::chrono::milliseconds(600000)
+#endif  // BIGTABLEINSTANCEADMIN_DEFAULT_MAXIMUM_RETRY_PERIOD
+
+#ifndef BIGTABLEINSTANCEADMIN_DEFAULT_MAXIMUM_DELAY
+#define BIGTABLEINSTANCEADMIN_DEFAULT_MAXIMUM_DELAY \
+  std::chrono::milliseconds(60000)
+#endif  // BIGTABLEINSTANCEADMIN_DEFAULT_MAXIMUM_DELAY
+
+const struct RPCPolicyDefaults kBigtableInstanceAdminLimits = {
+    BIGTABLEINSTANCEADMIN_DEFAULT_INITIAL_DELAY,
+    BIGTABLEINSTANCEADMIN_DEFAULT_MAXIMUM_RETRY_PERIOD,
+    BIGTABLEINSTANCEADMIN_DEFAULT_MAXIMUM_DELAY,
+};
+
+// Interface name: "google.bigtable.admin.v2.BigtableTableAdmin"
+// Define the defaults using a pre-processor macro, this allows
+// the application developers to change the defaults for their
+// application by compiling with different values.
+
+#ifndef BIGTABLETABLEADMIN_DEFAULT_INITIAL_DELAY
+#define BIGTABLETABLEADMIN_DEFAULT_INITIAL_DELAY std::chrono::milliseconds(100)
+#endif  // BIGTABLETABLEADMIN_DEFAULT_INITIAL_DELAY
+
+#ifndef BIGTABLETABLEADMIN_DEFAULT_MAXIMUM_RETRY_PERIOD
+#define BIGTABLETABLEADMIN_DEFAULT_MAXIMUM_RETRY_PERIOD \
+  std::chrono::milliseconds(600000)
+#endif  // BIGTABLETABLEADMIN_DEFAULT_MAXIMUM_RETRY_PERIOD
+
+#ifndef BIGTABLETABLEADMIN_DEFAULT_MAXIMUM_DELAY
+#define BIGTABLETABLEADMIN_DEFAULT_MAXIMUM_DELAY \
+  std::chrono::milliseconds(60000)
+#endif  // BIGTABLETABLEADMIN_DEFAULT_MAXIMUM_DELAY
+
+const struct RPCPolicyDefaults kBigtableTableAdminLimits = {
```

Comment:
If my suggestion above makes sense, this would become:

```C++
RPCPolicyParameters const kBigtableTableAdminDefaults{
```

Reply:
fixed.